app/models/book.rb

```diff
@@ -0,0 +1,16 @@
+class Book < ActiveRecord::Base
+  # default_scope { order(published_date: :desc) }
+  self.per_page = 25
+  
+  def self.by_times_purchased
+    order(times_purchased: :desc)
+  end
+
+  def self.search(search)
+    if search
+      Book.where("title LIKE ?", "%#{search}%")
+    else
```

Comment:
I don't think you'll ever hit this branch because it's called in the controller only if `params[:search]` is present. This `else` part can be removed.